method: org.apache.fop.apps.FOPException.getRootException:1281071440
317709189:this := @this: org.apache.fop.apps.FOPException
254836959:result = virtualinvoke this.<org.apache.fop.apps.FOPException: java.lang.Exception getException()>()
156696397:$z0 = result instanceof org.xml.sax.SAXException
781393157:if $z0 == 0 goto $z0 = result instanceof java.lang.reflect.InvocationTargetException
7273279:$r0 = (org.xml.sax.SAXException) result
69166454:result = virtualinvoke $r0.<org.xml.sax.SAXException: java.lang.Exception getException()>()
800723752:$z0 = result instanceof java.lang.reflect.InvocationTargetException
1631567762:if $z0 == 0 goto $r2 = virtualinvoke this.<org.apache.fop.apps.FOPException: java.lang.Exception getException()>()
1329670080:$r1 = (java.lang.reflect.InvocationTargetException) result
608482853:result = virtualinvoke $r1.<java.lang.reflect.InvocationTargetException: java.lang.Throwable getTargetException()>()
410637615:$r2 = virtualinvoke this.<org.apache.fop.apps.FOPException: java.lang.Exception getException()>()
1721437494:if result == $r2 goto return null
832096246:return result
1548490201:return null
